lottie/src/main/java/com/airbnb/lottie/Lottie.java

```diff
@@ -0,0 +1,59 @@
+package com.airbnb.lottie;
+
+import android.content.Context;
+
+import com.airbnb.lottie.network.DefaultFetcher;
+import com.airbnb.lottie.network.NetworkFetcher;
+import com.airbnb.lottie.network.Fetcher;
+import com.airbnb.lottie.network.NetworkCache;
+import com.airbnb.lottie.network.CacheProvider;
+
+import java.io.File;
+
+import androidx.annotation.NonNull;
+
+public class Lottie {
+
+  private static Fetcher fetcher;
+  private static CacheProvider cacheDirCacheProvider;
+
+  private static volatile NetworkFetcher networkFetcher;
+  private static volatile NetworkCache networkCache;
+
+  public static void initialize(@NonNull final LottieConfig lottieConfig) {
+    fetcher = lottieConfig.networkFetcher;
+    cacheDirCacheProvider = lottieConfig.cacheDirCacheProvider;
+  }
+
+  @NonNull
+  public static NetworkFetcher networkFetcher(@NonNull Context context) {
```

Comment:
Can you move these to `L.java`. `L` is where Lottie keeps static configuration for its own internal use. (Initialize should remail here though)

Reply:
done